samples/TagHelperSample.Web/Views/Shared/_Layout.cshtml

```diff
@@ -1,9 +1,9 @@
 ﻿<!DOCTYPE html>
 <html>
 <head>
-    <link href="~/blank.css" rel="stylesheet" />
-    <link href="~/site.min.css" rel="stylesheet"
-          asp-fallback-href="~/site.css"
+    <link href="~/css/sub/blank.css" rel="stylesheet" data-test="value" />
+    <link href="~/site.min.css" rel="stylesheet" data-test="value"
+          asp-fallback-href-include="**/blank.css,**/site.css"
           asp-fallback-test-class="fallback-test"
```

Comment:
Resolved globbed results are de-duped with static href, so you won't get dupes. You can omit href altogether and just rely on globbing. Fallback is really orthogonal, it just applies a test and if it fails it loads the fallback. Whether you actually referenced anything prior to the test doesn't really matter.